### DISCUSSION TITLE
Trying to match results after updates - Original installed version 43cca93

### DISCUSSION THREAD

**Discussion by creeduk** [Mikubill/sd-webui-controlnet#430] [Q&A]
I started with this build - https://github.com/Mikubill/sd-webui-controlnet/commit/43cca930e6e1c469a0f249ee925707b272c6d21d

Loved it had a few memory issues if I pushed it too far but running and on 2gb of VRAM so I was very happy. I updated recently to these versions (Friday, Saturday and yesterday) fe40f20, 0c45b5 and currently 3a1c5b9.

I noticed some burn out on some of my images like over configured, so I tried to replicate my earlier testing. One example was the dog sketch from the provided sample folder. I posted on reddit as well but figured it was time to ask here as maybe somebody knows a setting or if the code changed to a point where old results are no longer possible?

![dog_rel](https://user-images.githubusercontent.com/126584565/221936681-ba58c970-3427-47ac-81ab-38516b1a8e59.jpg)
Sample used as sketch:

**Results from 43cca93:**

![01229-925894899-highly detailed, masterpiece, dog,](https://user-images.githubusercontent.com/126584565/221937118-73bec415-2969-4c3e-abd7-971b53a43f37.png)

_highly detailed, masterpiece, dog,
Negative prompt: longbody, lowres, bad anatomy, bad hands, missing fingers, extra digit, fewer digits, cropped, worst quality, low quality
Steps: 20, Sampler: DPM++ SDE, CFG scale: 7, Seed: 925894899, Size: 512x512, Model hash: 629ddef959, Model: inkpunk-diffusion-v1, ControlNet Enabled: True, ControlNet Module: scribble, ControlNet Model: controlnetPreTrained_scribbleV10 [c508311e], ControlNet Weight: 1_

**Results from 3a1c5b9:**

![01582-925894899-highly detailed, masterpiece, dog,](https://user-images.githubusercontent.com/126584565/221946455-228595d8-82c5-4dab-9e10-7c0b5e01f24b.png)

_highly detailed, masterpiece, dog,
Negative prompt: longbody, lowres, bad anatomy, bad hands, missing fingers, extra digit, fewer digits, cropped, worst quality, low quality
Steps: 20, Sampler: DPM++ SDE, CFG scale: 7, Seed: 925894899, Size: 512x512, Model hash: 629ddef959, Model: inkpunk-diffusion-v1, ControlNet Enabled: True, ControlNet Module: scribble, ControlNet Model: controlnetPreTrained_scribbleV10 [c508311e], ControlNet Weight: 1, ControlNet Guidance Start: 0, ControlNet Guidance End: 1_

Tried numerous ways to match I would assume guidance start 0, end 1 and weight 1 but none of my test subjects match. I can confirm  model has not changed and if I test old generations without controlnet they are exact (tested in PS with difference mode)

**Comment by ljleb** [comment]
Not sure, but maybe try toggling the "Only use mid-control when inference" option in the settings? May be related to #373

**Reply by creeduk** [reply]
I saw that post a few days ago when I had the image burn out and at first I thought oh great the colors are fixed, but then I realized it was not following my sketch or any outlines at all. Not sure how that works it is like having a really low weight from what I could tell or a low cfg.

Before all the new settings and sliders I was wondering what were all the default hardcoded values in 43cca93?

**Comment by Mikubill** [comment]
Tested with https://github.com/Mikubill/sd-webui-controlnet/commit/96f2e1c12bab80488c44c5595c6efc95222cee03

| 3a1c5b9 | 96f2e1c |
|:-------------------------:|:-------------------------:|
|  <img width="256" alt="" src="https://user-images.githubusercontent.com/31246794/222016186-bffef2c9-0a57-49a7-8139-76a9fb8cddcd.png"> | <img width="256" alt="" src="https://user-images.githubusercontent.com/31246794/222016193-ca7d4ef1-85b9-4410-bd6a-2c1bb97ec6b0.png"> |

This change was introduced by https://github.com/Mikubill/sd-webui-controlnet/pull/350 10 hours ago - already reverted.

**Reply by creeduk** [reply]
I confirm that the weekend update and todays update create same output with only a tiny difference on the collar stiches. I still have different output from 43cca93. I plan to temp roll back to double check. Did some fixes that maybe make the function better change the output? Sometimes I know we have to move on and the old cannot be recreated.

![01642-925894899-highly detailed, masterpiece, dog,](https://user-images.githubusercontent.com/126584565/222274544-a8090983-e3a3-49db-95ce-18189e5d500a.png)

Thank you for checking and providing feedback. I know consistency is a huge challenge with this technology.
